packages/CalloutParagraph/CalloutParagraph.jsx

```diff
@@ -59,17 +61,46 @@ const TextWrapper = styled.div`
       padding-left: ${props => paddingValue.mobile[props.spacing]};
       padding-right: ${props => paddingValue.mobile[props.spacing]};
     }
+    > i {
+      margin-right: 1rem;
+      position: absolute;
+      left: 10px;
+    }
+    > span {
+      display: inline-block;
+      margin-left: ${props => props.margin};
+    }
   }
 `
 
 /**
  * @version ./package.json
  * @visibleName CalloutParagraph (beta)
  */
-const CalloutParagraph = ({ children, spacing, roundedCorners, compact, ...rest }) => {
+const CalloutParagraph = ({ children, spacing, roundedCorners, compact, icon, ...rest }) => {
+  const Icon = DecorativeIcons[icon]
+  const iconMargin = (spacingLevel, hasIcon) => {
+    if (hasIcon) {
+      if (spacingLevel === 'compact') {
+        return '1.5rem'
+      }
+      return '1rem'
+    }
+    return undefined
+  }
+
   return (
-    <TextWrapper spacing={spacing} roundedCorners={roundedCorners} compact={compact}>
-      <Paragraph {...safeRest(rest)}>{children}</Paragraph>
+    <TextWrapper
+      spacing={spacing}
+      roundedCorners={roundedCorners}
+      compact={compact}
+      icon={icon}
```

Comment:
Why is the icon prop being passed to TextWrapper.  It doesnt look like its being used

Reply:
Good question! This has been updated :) Thanks for catching that!